ampd/src/broadcaster/accounts.rs

```diff
@@ -16,7 +16,7 @@ pub enum Error {
     MalformedResponse,
 }
 
-pub async fn account<T>(mut client: T, address: &TMAddress) -> Result<BaseAccount, Error>
+pub async fn account<T>(client: &mut T, address: &TMAddress) -> Result<BaseAccount, Error>
```

Comment:
huh, that was a strange decision to have an owned parameter previously

Reply:
yup it was.